osu.Game/Screens/Play/InGameLeaderboard.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Users;
+
+namespace osu.Game.Screens.Play
+{
+    public class InGameLeaderboard : CompositeDrawable
```

Comment:
`Leaderboard`

Reply:
will become confusing as hell with the leaderboard component, better to add an `InGame` before the components to mark them as in-game components